pkg/kubelet/kuberuntime/kuberuntime_container_test.go

```diff
@@ -237,6 +237,9 @@ func TestToKubeContainerStatus(t *testing.T) {
 // TestToKubeContainerStatusWithResources tests the converting the CRI container status to
 // the internal type (i.e., toKubeContainerStatus()) for containers that returns Resources.
 func TestToKubeContainerStatusWithResources(t *testing.T) {
+	if goruntime.GOOS == "windows" {
+		t.Skip("InPlacePodVerticalScaling is not currently supported on Windows.")
+	}
```

Comment:
The individual test cases below have a `skipOnWindows` option. If we're blanket skipping this whole test, can we remove that per-testcase version?

Reply:
let me check the list of tests we are skipping against what is currently failing.
If it is only a few  additional tests that need skipping i'll update the skipOnWindows, otherwise I can skip them all and drop the skipOnWindows per-testcase

Reply:
I removed the `skipOnWindows` support here.
thanks!

Reply:
I was expecting this to be fixed in the other direction - skipping the individual cases that don't makes sense. We're already returning a different set of runtime resources depending on the OS

Reply:
Alternatively, if all of these are failing on windows, I'd suggest filing an issue to fix it, and add a TODO here linking the issue to remove the skip once it's fixed.  This test isn't specific to in-place resize.

Reply:
EDIT: never mind the previous (deleted) comments - looking at the code I see the resource conversion is gated on `InPlacePodVerticalScaling` and only performed for linux.